Add install info to GIS applications (#1723)
CSCfi/csc-user-guide: docs/apps/opendronemap.md

`OpenDroneMap Authors, ODM - A command line toolkit to generate maps, point clouds, 3D models and DEMs from drone, balloon or kite images. OpenDroneMap/ODM GitHub Page 2020; https://github.com/OpenDroneMap/ODM`

> ## Acknowledgement

Reviewer: ```suggestion
```

## Acknowledgement

Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".

## Installation

> Opendronemap was installed on Puhti with Singularity using the [Docker image provided by Opendronemap](https://hub.docker.com/r/opendronemap/odm). The container was then wrapped with [Tykky](../computing/containers/tykky.md).

Reviewer: ```suggestion
OpenDroneMap was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [OpenDroneMap Docker image from Dockerhub provided by OpenDroneMap community](https://hub.docker.com/r/opendronemap/odm). 
```
